<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmAddFriendDG</class>
 <widget class="QDialog" name="ConfirmAddFriendDG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="m_pShowAddFriendMsg">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pAgreeBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pRejectBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
